styleguide/components/41-organisms/timeline/_timeline.scss

```diff
@@ -308,25 +367,51 @@ dl.timeline {
             width: calc((100% / 1) - 2vw);
             margin: 1vw;
 
+            .gallery-link {
+              padding-bottom: 2.3em;
+            }
+
             // only two images
-            &:not(.image-gallery__show-more):nth-of-type(n + 3) {
+            &:not(.image-gallery__show-more):nth-of-type(n + 2) {
               display: none;
             }
           }
 
           .image-gallery__show-more {
-            padding-bottom: calc(((100% / 1) - 2vw) / 1.6);
+
+            // padding-bottom: calc(((100% / 1) - 2vw) / 1.6);
+            padding-bottom: 2.3em;
+
+            &::before {
+              background-color: transparent;
+            }
+
+            .show-more__content {
+              @include theme('color', 'color-primary', 'link-color');
+
+              display: inline-block;
+              margin-top: .3rem;
+              text-align: right;
+
+              .show-more__total {
+                display: none;
+              }
+            }
           }
 
           @include mobile {
             max-width: 270px;
           }
 
           @include phablet {
-            max-width: calc(100% + 2vw);
+            max-width: calc(270px * 2 + 2vw);
```

Comment:
Why are you using px here instead of rems?

Reply:
Because they have no relation to the font size, they should not grow or shrink when the user adjusts font-size.

This is set to keep the images small enough in relation to the screen (px).